jit.en.vtt

```diff
@@ -2481,7 +2481,7 @@ it's on my fork of CPython.
 [Brandt] That's the URL.
 
 32:40.195 --> 32:43.000
-So it's the `justin` branch of brandtbucher/cpython
+So it's the justin branch of brandtbucher/cpython
```

Comment:
This is marked up on purpose, as it can look like an unfamiliar adjective.
It would be typeset in monospace, so it uses Markdown-style quotes for that.

Reply:
Maybe just use "quotes", since the captions don't render in Markdown?

Reply:
```suggestion
So it's the “justin” branch of brandtbucher/cpython
```